Eastern-Research-Group/csb-rebate-forms-app
# Feature/implement pre 2023 prf release

## Related Issues:
* CSBAPP-325

## Main Changes:
* Major update to the wrapping application application, primarily to support showing 2023 FRF submissions CSB Rebate Ids, and BAP statuses for users. This is a quick release to support the "Edits Requested" workflow (full release notes will be included in the upcoming v5 release).
* Temporarily removes all references to the 2023 PRF and change request form (to be re-enabled in the v5 release).

**NOTE:** normally we'd merge PR's into the `develop` branch first, but work for the 2023 PRF and BAP de-duplication prototype form is ongoing on the dev site, so we're merging this "out of order" directly to the `staging` branch, so testing can occur on the `staging` site for the v4.1 release, without interrupting development that's occurring on the dev site for the v5 release.

## Steps To Test:
1. Navigate to the dashboard
2. Ask the BAP team to change an existing 2023 FRF submission's status to "Edits Requested"
3. Ensure the 2023 FRF submission period is closed, but the submission remains editable
4. Ask the BAP team to change an existing 2023 FRF submission's status to "Accepted"
5. Ensure the "New Payment Request" button does not appear below the selected/accepted FRF submission.
6. Ensure there are no references to the 2023 PRF or change request form.

## TODO:
- [ ] Re-enable all 2023 PRF and Change Request form code before the v5 release
